rules-documentation/logsource-guides/windows/category/process_creation.md

```diff
@@ -0,0 +1,179 @@
+GUID: {54849625-5478-4994-a5ba-3e3b0328c30d}
+Channel: Security
+EventID: 4688
+```
+
+```yml
+Provider: Microsoft-Windows-Sysmon
+GUID: {5770385f-c22a-43e0-bf4c-06f5698ffbd9}
+Channel: Microsoft-Windows-Sysmon/Operational
+EventID: 1
+```
+
+## Logging Setup
+
+This section described how to setup logging in your environment
+
+### Microsoft Windows Security Auditing
+
+#### Process Creation
+
+- Subcategory GUID: `{0CCE922B-69AE-11D9-BED3-505054503030}`
+- Provider: `Microsoft Windows Security Auditing`
+- Channel: `Security`
+- Event Volume: `High`
+- EventID(s):
+  - `4688`
+
+If you're using `gpedit.msc` or similar you can enable logging for this category by following the structure below
+
+```yml
+- Computer Configuration
+    - Windows Settings
+        - Security Settings
+            - Advanced Audit Policy Configuration
+                - System Audit Policies - Local Group Policy Object
+                    - Detailed Tracking
+                        - Audit Process Creation
+                            - Success and Failure
+```
+
+Alternatively you can enable logging via `auditpol` using the following command(s):
+
+```powershell
+# Enable Success audit Only
+auditpol /set /subcategory:{0CCE922B-69AE-11D9-BED3-505054503030}, /success:enable
+
+# Enable both Success and Failure auditing
+auditpol /set /subcategory:{0CCE922B-69AE-11D9-BED3-505054503030}, /success:enable /failure:disable
+```
+
+If you want to learn more about this sub-category. You can do so via MSDN - [Learn More](https://learn.microsoft.com/en-us/windows/security/threat-protection/auditing/audit-process-creation)
+
+#### Include Command-Line In Process Creation Events
+
+If you're using `gpedit.msc` or similar you can enable logging for this category by following the structure below
+
+```yml
+- Computer Configuration
+    - Administrative Templates
+        - System
+            - Audit Process Creation
+                - Include Command Line In Process Creation Events
+```
+
+### Microsoft-Windows-Sysmon
+
+#### Process Creation
+
+- Provider: `Microsoft-Windows-Sysmon`
+- Channel: `Microsoft-Windows-Sysmon/Operational`
+- Event Volume: `High`
+- EventID(s):
+  - `1`
+
+To configure Sysmon process creation events you can follow the instructions below
+
+- Download [Sysmon](https://learn.microsoft.com/en-us/sysinternals/downloads/sysmon)
+- Install Sysmon using an appropriate configuration. The configuration must include a `<ProcessCreate>` element. We recommend the following configuration [sysmonconfig-export.xml](https://github.com/Neo23x0/sysmon-config/blob/master/sysmonconfig-export.xml).
+
+```powershell
+sysmon -i /path/to/config
+```
+
+## Event Fields
+
+Note: For rules using this category in SIGMA. Know that there is a mapping between `Sysmon EID 1` fields and `Microsoft Windows Security Auditing EID: 4688`. While you can use the fields of `EID 4688` it's best to use the Sysmon ones.
+
+### Provider: Microsoft Windows Security Auditing / EventID: 4688
```

Comment:
```suggestion
### Provider: Microsoft Windows Security Auditing / EventID: [4688](https://www.ultimatewindowssecurity.com/securitylog/encyclopedia/event.aspx?eventid=4688)
```
As an example. If you like you could add it everywhere.

Reply:
I don't like using UltimateWindowsSecurity right now. In future updates maybe we'll add it too. For now, I need to double-check check information and it might take a while.

Reply:
In future version I'm planing to use information form there to enrich events too.